exercises/conversions/try_from_into.rs

```diff
@@ -66,71 +63,98 @@ mod tests {
     use super::*;
 
     #[test]
-    #[should_panic]
     fn test_tuple_out_of_range_positive() {
-        let _ = Color::try_from((256, 1000, 10000)).unwrap();
+        assert!(Color::try_from((256, 1000, 10000)).is_err());
     }
     #[test]
-    #[should_panic]
     fn test_tuple_out_of_range_negative() {
-        let _ = Color::try_from((-1, -10, -256)).unwrap();
+        assert!(Color::try_from((-1, -10, -256)).is_err());
+    }
+    #[test]
+    fn test_tuple_sum() {
+        assert!(Color::try_from((-1, 255, 255)).is_err());
     }
     #[test]
     fn test_tuple_correct() {
-        let c: Color = (183, 65, 14).try_into().unwrap();
+        let c: Color = match (183, 65, 14).try_into() {
+            Ok(color) => color,
+            Err(_) => Color {
+                red: 0,
+                green: 0,
+                blue: 0,
+            },
+        };
         assert_eq!(c.red, 183);
         assert_eq!(c.green, 65);
         assert_eq!(c.blue, 14);
     }
-
     #[test]
-    #[should_panic]
     fn test_array_out_of_range_positive() {
-        let _: Color = [1000, 10000, 256].try_into().unwrap();
+        let c: Color = [1000, 10000, 256].try_into();
+        assert!(c.is_err());
     }
     #[test]
-    #[should_panic]
     fn test_array_out_of_range_negative() {
-        let _: Color = [-10, -256, -1].try_into().unwrap();
+        let c: Color = [-10, -256, -1].try_into();
+        assert!(c.is_err());
+    }
+    #[test]
+    fn test_array_sum() {
+        let c: Color = [-1, 255, 255].try_into();
+        assert!(c.is_err());
     }
     #[test]
     fn test_array_correct() {
-        let c: Color = [183, 65, 14].try_into().unwrap();
+        let c: Color = match [183, 65, 14].try_into() {
+            Ok(color) => color,
+            Err(_) => Color {
+                red: 0,
+                green: 0,
+                blue: 0,
+            },
+        };
         assert_eq!(c.red, 183);
         assert_eq!(c.green, 65);
         assert_eq!(c.blue, 14);
     }
-
     #[test]
-    #[should_panic]
     fn test_slice_out_of_range_positive() {
         let arr = [10000, 256, 1000];
-        let _ = Color::try_from(&arr[..]).unwrap();
+        assert!(Color::try_from(&arr[..]).is_err());
     }
     #[test]
-    #[should_panic]
     fn test_slice_out_of_range_negative() {
         let arr = [-256, -1, -10];
-        let _ = Color::try_from(&arr[..]).unwrap();
+        assert!(Color::try_from(&arr[..]).is_err());
+    }
+    #[test]
+    fn test_slice_sum() {
+        let arr = [-1, 255, 255];
+        assert!(Color::try_from(&arr[..]).is_err());
     }
     #[test]
     fn test_slice_correct() {
         let v = vec![183, 65, 14];
-        let c = Color::try_from(&v[..]).unwrap();
+        let c = match Color::try_from(&v[..]) {
+            Ok(color) => color,
+            Err(_) => Color {
+                red: 0,
+                green: 0,
+                blue: 0,
+            },
+        };
```

Comment:
Same as upper checking against with `assert_eq!` an expected value wrapped in a `std::Result` will be more idiomatic :heart: .